operator/templates/generate-cassandra-yaml.yaml

```diff
@@ -1,10 +1,18 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Name }}-cassandra-yaml
+  name: {{ .Name }}-generate-cassandra-yaml
   namespace: {{ .Namespace }}
 data:
-  cassandra.yaml: |
+  generate-cassandra-yaml.sh: |
+    #!/usr/bin/env bash
+
+    set -euxo pipefail
+
+    readonly truststore_password=$(cat /etc/cassandra/truststore/truststore_password)
+    readonly keystore_password=$(cat /etc/cassandra/truststore/keystore_password)
+
+    cat <<EOF >> /etc/cassandra/cassandra.yaml
```

Comment:
we want to append here or overwrite? I'm thinking about restarts caused by health check fails etc

Reply:
Overwrite. Oh, this is appending, that's a mistake. Will fix this.